src/redux/actions/beerActions.js

```diff
@@ -46,3 +51,28 @@ export function getFavoriteBeer() {
     }
   };
 }
+
+export function searchBeers(title) {
+  return async (dispatch) => {
+    dispatch(getBeerStarted());
+
+    try {
+      const res = await axios.get(process.env.REACT_APP_BEER_URL);
+      const titleLowerCase = title.toLowerCase();
+      const searched = res.data.filter((beer) => beer.name.toLowerCase().includes(titleLowerCase));
```

Comment:
You should create a request to search particular beers instead of fetching all of them and filtering needed.
Please use beer_name as a query param

Consider using `lodash.debounce` library to avoid too much call as user types